ocs_ci/helpers/cephfs_stress_helpers.py

```
@@ -728,71 +773,120 @@ def _check_pods_health():
+                    baseline_count = current_baseline.get(key, 0)
+                    delta = restart_count - baseline_count
+
+                    if delta < 0:
+                        # restartCount went backwards — the pod was recreated with the
+                        # same name and its counter reset to 0.
+                        delta = restart_count
+                        new_baseline[key] = 0
+                        logger.warning(
+                            f"Pod {pod_name} container {container_name}: restartCount "
+                            f"dropped from baseline {baseline_count} to {restart_count} "
+                            f"— pod likely recreated, resetting baseline"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)